app/config/config.yml

```diff
@@ -191,6 +197,20 @@ doctrine:
 doctrine_migrations:
     dir_name: "%kernel.root_dir%/DoctrineMigrations"
 
+# Swiftmailer Configuration
+swiftmailer:
+    spool:     { type: memory }
+    transport: "%mailer_transport%"
+    host:      "%mailer_host%"
+    username:  "%mailer_user%"
+    password:  "%mailer_password%"
+    # using an env var throws an error
+    # auth_mode: "%mailer_auth_mode%"
+    auth_mode: cram-md5
```

Comment:
Hmm.  Hard-coding this seems bad.  I wonder if there is another way to get the value set that will work.